packages/apps/public/locales/en/apps.json

```diff
@@ -1,5 +1,6 @@
 {
   "Accounts": "Accounts",
+  "Connect": "Connect",
```

Comment:
It would prefer this under settings

(a) we already have it under accounts (when you have nothing)
(b) it is certainly not something that the user would visit often

Reply:
Connecting to an extension is basically the first step to use the dapp fully.
Most dapps that I know have a Connect button somewhere in the right top corner.
I didn't put it there cause there are other blocks there now, so decided next to settings would be best.
Most crypto users are very familiar with this pattern and doing it differently here would not be to our benefit.

Simple possible scenario: User enters polkadot-js/apps, looks for Connect, can't find it.
The page that is opened by default is "explorer", which for any new person is just a gibberish of random numbers and letters, quite useless, "I want to see ma balance, sers, not the matrix!", "Tom was right, this polkadot thing is for rocket scientists, I'm out" (I've omitted a lot of swearing from those quotes).

And that is how we lose new people. Since we are not a closed-door invite-only club, on the contrary my team's mission is to onboard the world and not just rocket scientists.
So hiding stuff that is important IMO not the way. Putting a large text in accounts hoping that user actually opens the accounts and reads it is also not the best thing ever I think.

Take into account that with this PR user needs to connect to each extension himself for the first time, this prevents multi-popup issue, this also lets user decide what extension he wants to use/install/connect.

Reply:
Ok, it is not hiding stuff - we are no hiding stuff since we first expect people to install stuff and enable stuff individually, instead of whatever extensions he/she has installed "just working". There is no thinking. When he/she has an extensions, the accounts show up.

If he/she has nothing, they will be prompted on the accounts page (based on the extensions that have actually listed themselves there).

![image](https://user-images.githubusercontent.com/1424473/187702690-6074e178-35a1-4bc7-86dc-9c4e3b2fcbec.png)

The current approach, as imaged here, adds friction - install extension, add account, then randomly connect it. The connection should "just work" when it "is there".